src/category_theory/instances/topological_spaces.lean

```diff
@@ -105,10 +105,10 @@ end Top
 
 variables {X : Top.{u}}
 
-instance : small_category (opens X) := by apply_instance
+instance : category (opens X) := category_theory.preorder_category (opens X)
```

Comment:
```suggestion
instance : opens.category (opens X) := category_theory.preorder_category (opens X)
```
Travis is reporting a conflict. I think it is about this name.

Reply:
I’ve just decided this is a crappy PR, anyway.

The problem is that because the morphisms are in `Prop`, Lean hides definitions inside auxiliary `_proof_k` fields, which you can’t unfold, and simp lemmas can’t see.

Reply:
OK. Another problem is that you would no longer be able to talk about (co)limits indexed by a preorder (without using "`vlift`", which doesn't exist yet).